src/WebJobs.Script.Abstractions/WebJobs.Script.Abstractions.csproj

```diff
@@ -13,6 +14,7 @@
     <RootNamespace>Microsoft.Azure.WebJobs.Script.Abstractions</RootNamespace>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)key.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <Deterministic>True</Deterministic>
```

Comment:
What is this for?

Reply:
Using deterministic flag: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/code-generation#deterministic will ensure we produce same hash ! @fabiocav  recommended this!

Reply:
Oh, nice! Should this go in common.props? Or would that add it to too many other projecs?

Reply:
Haven't tried it out. Will try it now.

Reply:
Done. common.props seems to be okay.